src/content/getting-started/adding-content.md

```diff
@@ -84,6 +84,59 @@ sections:
     
 ```
 
+#### Supporting live-editing of ngdocs examples
+If you use `swanky-processor-ngdocs` as a preprocessor for `ngdoc` comments in your source code,
+you can edit the code for the example components directly in your browser. This is known as live-editing.
+To enable this feature, one of your additional scripts will need to import `swanky-processor-ngdocs/src/bootstrap/angular.bootstrap.js` for
+Angular components, or `swanky-processor-ngdocs/src/bootstrap/react.bootstrap.js` for React components. 
+Other kinds of components can be supported too.
+
+##### Angular Example
+```js
+/* File: path/to/script-1.js */
+import ngDocsBootstrap from 'swanky-processor-ngdocs/src/bootstrap/angular.bootstrap';
+
+// This line MUST contain a static-path (rather than a variable) so that Webpack can determine the list of files to require() at compile-time.
+const modulesToRequire = require.context('../../content/angular-components/', true, /^.*\/index.js$/);
+
+// Load all of the Angular modules that you can find then boot the app.
+function requireAll(requireContext) {
+  return requireContext.keys().map(requireContext);
+}
+
+const modules = requireAll(modulesToRequire);
+
+// Assume module.default is the module name (a string)
+const moduleMap = modules.reduce((acc, mod) => Object.assign(acc, {[mod.default]: mod}), {});
+
+ngDocsBootstrap(moduleMap);
+```
+
+##### React Example
+```js
+/* File: path/to/script-1.js */
+import reactBootstrap from 'swanky-processor-ngdocs/src/bootstrap/react.bootstrap';
```

Comment:
I think it would be nicer to be able import bootstrap files as follows:

```
import { reactBootstrap } from 'swanky-processor-ngdocs';
```

What are your thoughts? Could be something I adjust later.